packages/web-components/src/search-widget/client/components/google-map.svelte

```diff
@@ -0,0 +1,61 @@
+<script lang="typescript">
+  import { onMount, onDestroy } from 'svelte'
+  import * as SearchWidgetStore from '../core/store'
+  import { getLatLng, createMarker } from '../../../common/utils/map-helper'
+  import { DEFAULT_CENTER, DEFAULT_ZOOM } from '../../../common/utils/constants'
+  import { showPropertiesMarker } from '../utils/google-map'
+  import { loader } from '../../../common/utils/loader'
+  
+  const searchWidgetStore = SearchWidgetStore.default
+
+  let storeInstance: SearchWidgetStore.SearchWidgetStore
+  let map: google.maps.Map
+  let mapElement: HTMLDivElement
+  let mapLoading: boolean = false
+
+  const unsubscribeSearchWidgetStore = searchWidgetStore.subscribe(store => {
+    if (map && storeInstance && storeInstance.properties !== store.properties) {
```

Comment:
This is a note to me - I am not sure this is the best solution but will try and come up with something better - leave for now!